CMGTools/TTHAnalysis/cfg/run_susyMultilepton_cfg.py

```diff
@@ -9,10 +9,23 @@
 #-------- LOAD ALL ANALYZERS -----------
 
 from CMGTools.TTHAnalysis.analyzers.susyCore_modules_cff import *
+from PhysicsTools.HeppyCore.framework.heppy_loop import getHeppyOption
 
-#-------- REDEFINE WHAT I NEED -----------
-
-SOS = False ## switch True to overwrite settings for SOS skim (N.B. default settings are those from multilepton preselection)
+#-------- SET OPTIONS AND REDEFINE CONFIGURATIONS -----------
+
+is50ns = getHeppyOption("is50ns",False)
+runData = getHeppyOption("runData",False)
+runDataQCD = getHeppyOption("runDataQCD",False)
+runFRMC = getHeppyOption("runFRMC",False)
+scaleProdToLumi = float(getHeppyOption("scaleProdToLumi",-1)) # produce rough equivalent of X /pb for MC datasets
+SOS = getHeppyOption("SOS",False) ## switch True to overwrite settings for SOS skim (N.B. default settings are those from multilepton preselection)
+saveSuperClusterVariables = getHeppyOption("saveSuperClusterVariables",False)
+removeSeparateJetCorrections = getHeppyOption("removeSeparateJetCorrections",False)
+doMETpreprocessor = getHeppyOption("doMETpreprocessor",False)
+doMETNoHF = getHeppyOption("doMETNoHF",True)
```

Comment:
this line may be tricky since running with `-o doMETNoHF=False` will cause `doMETNoHF` to be the string `'False'`, which evaluates to true in python.
Maybe `doMETNoHF =(getHeppyOption("doMETNoHF",True) != 'False')` instead ?